Themis/Models/Submission.swift

```diff
@@ -58,8 +58,9 @@ struct SubmissionForAssessment: Codable {
     // let results: [SubmissionResult]
 }
 
-struct GradingCriteria: Codable, Identifiable {
+struct GradingCriterion: Codable, Identifiable {
     var id: Int
+    var title: String
```

Comment:
This crashes the app for me. Apparently, I don't get a title field. I suppose this should be optional.

Reply:
For reference, here's what I get when starting a new assessment:
```keyNotFound(CodingKeys(stringValue: "title", intValue: nil), Swift.DecodingError.Context(codingPath: [CodingKeys(stringValue: "participation", intValue: nil), CodingKeys(stringValue: "exercise", intValue: nil), CodingKeys(stringValue: "gradingCriteria", intValue: nil), _JSONKey(stringValue: "Index 0", intValue: 0)], debugDescription: "No value associated with key CodingKeys(stringValue: \"title\", intValue: nil) (\"title\").", underlyingError: nil))```

Reply:
Oh, thank you, I will further explorate this.

Reply:
Can you check it again. I hope it should work now.

Reply:
<img width="806" alt="Bildschirm­foto 2022-12-01 um 16 23 17" src="https://user-images.githubusercontent.com/116847304/205091684-559550ac-48ec-4b73-a32d-1e9fa4a034a5.png">

Reply:
<img width="806" alt="Bildschirm­foto 2022-12-01 um 16 23 59" src="https://user-images.githubusercontent.com/116847304/205091818-09a8bc37-98b8-48ef-9c5f-6b33cb56867f.png">

Reply:
Now it works with and without a title.